packages/storybook/src/templates/mijn-omgeving-home/index.css

```
@@ -249,29 +285,20 @@
 }
 
 .todo-page-footer__spotlight {
-  --ams-page-footer-menu-link-color: var(--voorbeeld-color-white);
-  --ams-page-footer-menu-hover-color: var(--voorbeeld-color-gray-100);
-  --utrecht-heading-2-color: var(--voorbeeld-color-white);
-  --utrecht-heading-2-padding-start: 2em;
-  --utrecht-paragraph-color: var(--voorbeeld-color-white);
-  --utrecht-link-color: var(--voorbeeld-color-white);
-  --utrecht-link-hover-color: var(--voorbeeld-color-gray-100);
+  --ams-page-footer-menu-link-color: var(--todo-page-footer-menu-link-color);
+  --ams-page-footer-menu-hover-color: var(--todo-page-footer-menu-hover-color);
+  --utrecht-heading-4-color: var(--todo-page-footer-heading-4-color);
```

```suggestion
  --utrecht-heading-4-color: var(--todo-page-footer-heading-color);
```

Volgens mij maakt het inhoudelijk niet uit wat voor heading levels er in de footers zitten, we willen ze toch allemaal dezeflde kleur zetten.